### ISSUE TITLE
Extension initialization failed

### ISSUE THREAD

**Issue by Spring-long** [LeetCode-OpenSource/vscode-leetcode#784]
Before you submit an question we recommend you to check out the [DOCUMENT](https://github.com/LeetCode-OpenSource/vscode-leetcode/blob/master/README.md) first.

You can also find more information in:
- [TROUBLESHOOTING](https://github.com/LeetCode-OpenSource/vscode-leetcode/wiki/Troubleshooting)
- [FAQ](https://github.com/LeetCode-OpenSource/vscode-leetcode/wiki/FAQ)

## 💬 Questions and Help
After installing the plugin, click the Leetcode icon prompt Extension initialization failed. Please open output channel for details.

vscode: 1.66.0 (user setup)
node:v16.14.2
Node has been added to the environment variable
windows :No .lc in user directory
wsl: No .lc in user directory

leetcode output ：
/mnt/d/software/nodejs/node.exe
v16.14.2
/mnt/c/Users/spring/.vscode/extensions/leetcode.vscode-leetcode-0.18.0/node_modules/vsc-leetcode-cli/bin/leetcode
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'D:\mnt\c\Users\spring\.vscode\extensions\leetcode.vscode-leetcode-0.18.0\node_modules\vsc-leetcode-cli\bin\leetcode'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
/mnt/c/Users/spring/.vscode/extensions/leetcode.vscode-leetcode-0.18.0/node_modules/vsc-leetcode-cli/bin/leetcode
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'D:\mnt\c\Users\spring\.vscode\extensions\leetcode.vscode-leetcode-0.18.0\node_modules\vsc-leetcode-cli\bin\leetcode'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
Error: Command "wsl /mnt/d/software/nodejs/node.exe,/mnt/c/Users/spring/.vscode/extensions/leetcode.vscode-leetcode-0.18.0/node_modules/vsc-leetcode-cli/bin/leetcode,plugin,-i,company" failed with exit code "1".

**Comment by wiuuu666** [comment]
yeah i meet teh same problem, i had try the all way solve this problem. include(change path , no .lc) but i also can not open this shit extension. 
error:Extension initialization failed. Please open output channel for details.
